docs/manual-doc/data-types/numeric-types.md

```diff
@@ -0,0 +1,5 @@
+Two basic data types are supported by `Nebula` -- int64 & double.
+
+The range for int64 is [-9223372036854775808, 9223372036854775807]. There are no upper and lower bound for double.
```

Comment:
no upper or lower bound

Reply:
we 'll start another pr to fix those typos and minor errors.